src/components/CategoriesEmptyState/tests/CategoriesEmptyState.test.tsx

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+import { vi } from 'vitest';
+
+import { render, screen, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import CategoriesEmptyState from '..';
+
+vi.mock(`@clerk/nextjs`, () => ({
+  __esModule: true,
+  useUser: () => ({
+    user: {
+      id: `1`,
+    },
+  }),
+}));
+
+vi.mock(`axios`);
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe(`CategoriesEmptyState`, () => {
+  afterAll(() => {
+    vi.clearAllMocks();
+  });
+
+  afterEach(() => {
+    cleanup();
+    vi.resetAllMocks();
+  });
+
+  it(`should open create category modal when clicking on button "Cadastrar categoria"`, async () => {
+    mockedAxios.delete.mockResolvedValueOnce({ data: { deleted: false } });
+
+    render(<CategoriesEmptyState />);
+
+    const emptyStateNewCategoryBtn = screen.getByRole(`button`, {
+      name: `Cadastrar categoria`,
+    });
+
+    expect(emptyStateNewCategoryBtn).toBeInTheDocument();
```

Comment:
```suggestion
    const emptyStateNewCategoryBtn = screen.getByRole(`button`, {
      name: `Cadastrar categoria`,
    });
```
I think you don't need this, because if the system doesn't find the button it will crash your test